packages/composites/the-graph/src/methods/prices/prices.ts

```diff
@@ -0,0 +1,86 @@
+import { AdapterError, Validator, Requester, Logger } from '@chainlink/ea-bootstrap'
+import { Config, USD, USDT, WETH, UNISWAP } from "../../config"
+import { AdapterRequest, AdapterResponse } from "@chainlink/types"
+import { getUSDPriceInUSDT } from "./dataProvider"
+import { ethers } from "ethers"
+import { DexSubgraph } from "../../types"
+
+const customParams = {
+    baseCoinTicker: false,
+    quoteCoinTicker: false,
+    dex: false
+}
+
+export const execute = async (input: AdapterRequest, config: Config, provider: ethers.providers.Provider): Promise<AdapterResponse> => {
+    const validator = new Validator(input, customParams)
+    if (validator.error) throw validator.error
+    const jobRunID = validator.validated.id
+    const { baseCoinTicker = USD, quoteCoinTicker = USD, dex = UNISWAP } = validator.validated.data
```

Comment:
We should make sure we're not case sensitive here, so would recommend normalizing input to upper/lower-case before it's used.